todo_app/templates/index.html

```diff
@@ -22,7 +22,23 @@ <h2>Items</h2>
             </li> 
             
         {% endfor %} 
-          </ul>
+      </ul>
+    </div>
+    <div class="col-md-12">
+      <h2>Done Items</h2>
+      <ul class="list-group mb-4">
+        {% for item in view_model.done_items %} 
+          <li>{{ item.name }} - {{item.status}} 
+             {% if item.status != 'Done' %}
+             <form action= "/complete_Card" method="POST">
+              <input type="hidden" name="card_id" value="{{item.id}}" > 
+              <button type="submit" class="btn btn-primary">complete item</button>
+            </form>
+            {% endif %}
```

Comment:
This is the list of done items. So you know that all of them have a status of "Done". So you couldjust delete this whole if block and \<form>